.github/workflows/stale_bot.yml

```diff
@@ -2,28 +2,41 @@ name: Close stale issues and PRs # https://github.com/actions/stale
 on:
   workflow_dispatch:
   schedule:
-    - cron: '*/30 * * * *' # Every 30 mins
+    - cron: "*/30 * * * *" # Every 30 mins
 
 jobs:
   stale:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/stale@v8
         with:
-          # stale rules
-          days-before-stale: 60
+          # stale rules for PRs
           days-before-pr-stale: 7
           stale-issue-label: stale
-          stale-issue-message: "This issue has been automatically marked as stale because it has not had any activity for 60 days."
+          exempt-pr-labels: pinned,security,roadmap
 
-          # close rules
-          # days after being marked as stale to close
-          days-before-close: 30
-          close-issue-label: closed-stale
-          close-issue-message: This issue has been automatically closed it has not had any activity in 90 days."
           days-before-pr-close: 7
 
-          # exemptions
-          exempt-pr-labels: pinned,security,roadmap
+      - uses: actions/stale@v8
+        with:
+          # stale rules for high priority bugs
+          days-before-stale: 30
```

Comment:
should we put a warning on it, before the SLA expires? Rather than wait for it to happen then scramble to fix it

Reply:
Well the discord message is the warning - it doesn't have to be a scramble, it's just more drawing attention to it. I won't put any red siren emojis on the message 😄 

🚨 🚨 🚨 🚨 🚨 🚨 🚨 🚨 🚨 🚨 
Not this ^^

It'll be more: 

🐛 Fix me please! 🐞

Reply:
Makes sense! Also if we did that we'd probably need another one to say "ok this is actually past the deadline now". Happy to leave as is